apis/python/setup.py

```diff
@@ -22,10 +22,15 @@
 import sys
 
 import setuptools
-
-# import setuptools.command.build_ext
+import setuptools.command.build_ext  # FIXME: why was this commented out?
```

Comment:
This import is+was used below, so I'm not sure why [it was commented out](https://github.com/single-cell-data/TileDB-SOMA/commit/123cb4370057eb47c5657ca2f846336a86e6032e#diff-66aed70e28174320f1db4eaccedcab15d85d14b96fb0c2e9f03417e6fc07df92R26), any feedback?

Reply:
I thought with `import setuptools` above we could just call `setuptools.command.build_ext.build_ext` below without actually needing to also `import setuptools.command.build_ext`.